experiments/long_runs/land.jl

```diff
@@ -57,6 +57,10 @@ diagnostics_outdir = joinpath(root_path, "global_diagnostics")
 outdir =
     ClimaUtilities.OutputPathGenerator.generate_output_path(diagnostics_outdir)
 
+function mask_land_tendencies(Yₜ, land_mask)
```

Comment:
just a note that we should not need this, since @sriharshakandala is adding the option of a mask directly to the ClimaCore space.

Reply:
@kmdeck Indeed! This is simply to test your code with the best available dataset from the ETOPO2022 archives. You already have the required function in the code actually - the `ocean_mask_values` blocks